addons/im_livechat/models/mail_channel.py

```diff
@@ -85,6 +85,7 @@ def channel_info(self, extra_info=False):
                 if channel.livechat_operator_id:
                     channel_infos_dict[channel.id]['operator_pid'] = channel.livechat_operator_id.with_context(im_livechat_use_username=True).name_get()[0]
                 # add the anonymous or partner name
+                channel_infos_dict[channel.id]['livechat_website_visitor'] = channel._channel_get_livechat_website_visitor_info()
                 channel_infos_dict[channel.id]['correspondent_name'] = channel._channel_get_livechat_partner_name()
```

Comment:
I feel like `_channel_get_livechat_partner_name` should eventually disappear to be computed in JS, especially with the introduction of `_channel_get_livechat_website_visitor_info`.
Maybe for the livechat task.

Reply:
Indeed, I just keep it for the moment, but livechat task should definitely remove this method and `correspondent_name` altogether.